app/models/equipment.rb

```diff
@@ -1,5 +1,6 @@
 class Equipment < ApplicationRecord
   belongs_to :user
+  has_many :bookings
   validates :ad_name, presence: true
   validates :category, presence: true
```

Comment:
Perhaps, need to add something like that
`inclusion: { in: CATEGORY }` and add a `CATEGORY = [...]`
(I'll let @VincentMsi think about categories ... because i don't really know about audio equipment)